shape-blog-react/.babelrc

```diff
@@ -0,0 +1,58 @@
+{
+  "presets": ["@babel/preset-react", ["@babel/preset-env", {
+    "modules": false,
+    "targets": {
+      "browsers": ["last 2 versions"]
+    }
+  }]],
+  "plugins": [
+    "react-hot-loader/babel",
+    [
+      "@babel/plugin-transform-runtime",
+        {
+          "regenerator": true
+        }
+    ],
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-json-strings",
```

Comment:
A lot of these presets are not needed. I think we should try to scale back to stage-3 equivalent. We aren't using a lot of these plugins.